### PR TITLE
[SHARE-604][Fix] Don't accept changes from the past

### PR DESCRIPTION
**Pull request by chrisseto**

### CODE REVIEW

File: tests/share/disambiguation/test_creativework.py [hunk 1]
```diff
@@ -174,3 +174,88 @@ def test_no_merge_on_blank_value(self, Graph):
...
+        assert older_graph.nodes[0].change == {'title': 'Not Previously Changed'}
+
+    def test_no_timetraveling_many(self, Graph):
+        oldest_graph = ChangeGraph(Graph(
+            Publication(
+                id=1,
+                sparse=True,
+                is_deleted=True,
+                title='The first title',
+                description='The first language',
```
Comment (leb2dg) [inline tests/share/disambiguation/test_creativework.py]: Should this be 'The first description'?

File: share/change.py [hunk 2]
```diff
@@ -276,8 +276,27 @@ def change(self):
...
+
+                # Go back until we find a change that is older than us
+                for version in self.instance.versions.select_related('change').all():
+                    if not version.date_updated or date_updated > version.date_updated:
+                        break
+                    ignore_attrs.update(version.change.change.keys())
+
         attrs = {}
         for k, v in self.attrs.items():
+            if k in ignore_attrs:
```
Comment (leb2dg) [inline share/change.py]: Will `ignore_attrs` only contain the keys for the change that resulted in the current instance (not the new change) and the change for the previous version?

File: tests/share/disambiguation/test_creativework.py (hunk 1 above)
Comment (chrisseto) [inline tests/share/disambiguation/test_creativework.py]: Yes.

File: share/change.py (hunk 2 above)
Comment (chrisseto) [inline share/change.py]: It will contain the keys of all changes that have a `date_modified` > the one in the current change.

Comment (leb2dg) [inline share/change.py]: Ah got it 👍  (`date_updated` not `date_modified`?)

**Comment by leb2dg** [comment]
Looks good to me 💵

**Pull request merged**